src/OrbitGl/CaptureWindow.cpp

```diff
@@ -152,7 +154,10 @@ void CaptureWindow::LeftDown(int x, int y) {
   click_was_drag_ = false;
 
   if (time_graph_ == nullptr) return;
-  ref_time_click_ = time_graph_->GetTime(static_cast<double>(x) / viewport_.GetScreenWidth());
+
+  float world_x = viewport_.ScreenToWorld(Vec2i(x, y))[0];
+  ref_time_click_ = time_graph_->GetUsFromTick(time_graph_->GetTickFromWorld(world_x)) +
```

Comment:
This looks completely wrong, we shouldn't add GetMinTimeUs(). 

The conversions should go from
- World: World coordinates
- Tick: An unsigned number to count the timestamps who usually start running as soon as the computer is turned on.
- Us: Milliseconds since the capture start.

So, you should directly convert from World to Us.

This looks completely wrong (erasing min_time_us_).
```
double TimeGraph::GetUsFromTick(uint64_t time) const {
  return TicksToMicroseconds(capture_min_timestamp_, time) - min_time_us_;
}
```

Actually, UsFromTick and TickFromUs doesn't seem to be inverse at all. I'm surprised that this is working, but I guess we are always using this Us as relative references with another Us because I can't understand why is working.


Also, consider to add a GetUsFromWorld in timeline_info_interface and using it directly.

Reply:
All of this is a mess, but I didn't want to change the `GetXFromY` methods in this PR because I'm not sure what it affects. But in the end, `GetUsFromTick` returns the us relative to `GetMinTimeUs()`, and I need the absolute time here. This is equivalent to what `GetTime()` returned before.

We currently don't have a method to directly convert from world to us... and the question is, if I introduce it here, should it behave as `GetUsFromTick` (which does not include min_time - which is very confusing IMO), or should it behave "correctly", but then it's inconsistent. So I decided to not add additional methods, fix it with a mini refactor, but we should completely redo the `GetXFromY` separately.

Reply:
Not related to this PR but I think we should have a refactor to clean up the current code in `TimeGraph`.

When I read `TimeGraph::PanTime`
```cpp
void TimeGraph::PanTime(int initial_x, int current_x, int width, double initial_time) {
  double time_window_us = max_time_us_ - min_time_us_;
  double initial_local_time = static_cast<double>(initial_x) / width * time_window_us;
  double dt = static_cast<double>(current_x - initial_x) / width * time_window_us;
  double current_time = initial_time - dt;
  double min_time_us =
      std::clamp(current_time - initial_local_time, 0.0, GetCaptureTimeSpanUs() - time_window_us);
  double max_time_us = min_time_us + time_window_us;
  SetMinMax(min_time_us, max_time_us);
}
``` 
It takes me a long time to understand the code here. I feel like that
* `initial_x` and `current_x` should be change to `initial_screen_x` and `current_screen_x` if I understand correctly
* what this `width` refers to? do we always use `timeline_width` here
* why we have such a complex logic to compute the `min_time_us`,
at least from the above code, it is equivalent to 
```
  double min_time_us =
      std::clamp(initial_time- static_cast<double>(current_x) / width * time_window_us, 0.0, GetCaptureTimeSpanUs() - time_window_us);
```